style.css

```diff
@@ -103,3 +116,278 @@ p {
   margin-top: 2px;
   list-style: none;
 }
+
+/* works section */
+.works {
+  background-color: #ebebff;
+  padding: 0 24px;
+}
+
+.grid-container {
+  display: grid;
+  grid-auto-rows: minmax(552px, auto);
+}
+
+.grid-item {
```

Comment:
:red_circle:The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that you should lay out the cards elements of this work section using CSS FlexBox. Kindly Use CSS FlexBox for the cards elements. Kudos for using  CSS Grid to lay out an the wrok section container  :+1: :100: .
![Screenshot from 2023-01-14 21-08-06](https://user-images.githubusercontent.com/26736582/212492143-747caf52-0953-4ee7-bf32-083fe73cb1db.png)